CHANGELOG.rst

```diff
@@ -7,6 +7,17 @@ The format is based on `Keep a Changelog`_,
 and this project adheres to `Semantic Versioning`_.
 
 
+Version 0.3.16 (2022-10-13)
+---------------------------
+
+* Added: argument ``tmp_root`` to
+  ``audbackend.Backend.get_archive()`` and
+  ``audbackend.Backend.put_archive()``
+* Changed: require ``flake8 <5.0.0``
+* Fixed: fix return value type hint of
+  ``audbackend.Backend._get_file()``
```

Comment:
I think those two points are not really relevant for the changelog.

```suggestion
```

Or at least I would delete the last one, and update the second one to

```suggestion
* Changed: test installation requires ``flake8 <5.0.0``
```

Reply:
You are right, I removed them.